geode-protobuf-messages/src/main/proto/protocolVersion.proto

```diff
@@ -25,13 +25,13 @@ enum MinorVersions {
     CURRENT_MINOR_VERSION = 1;  // Protobuf implementation at initial release
 }
 
-message NewConnectionHandshake {
```

Comment:
Do you think it would be worthwhile to add a comment saying that we will need both fields to be populated and nonzero for the handshake to work?